go/oauth2l/main.go

```diff
@@ -65,22 +77,24 @@ func main() {
 	}
 
 	scopes := flag.Args()[1:]
```

Comment:
Is Go flag parsing compatible with Python?

Reply:
The command line flag will stay the same. But the installation flow is different from Python, for example:
```
go get PATH
go install
~/go/bin/oauth2l --jwt --json JSONPATH fetch scope
```

Reply:
I will make sure the command ordering is the same in the next PR.